src/Services/Air/Air.js

```diff
@@ -36,8 +36,11 @@ module.exports = (settings) => {
 
     book(options) {
       return service.airPricePricingSolutionXML(options).then((data) => {
+        const tau = options.tau && moment(options.tau).isValid()
```

Comment:
Will this work also?
```
const tauDate = moment(options.tau);
const tau = tauDate.isValid() ? tauDate.format() : moment().add(3, 'hours').format();
```

Reply:
But this will:
```javascript
const tauDate = moment(options.tau || null);
const tau = tauDate.isValid() ? tauDate.format() : moment().add(3, 'hours').format();
```